code/Player/Player.cs

```diff
@@ -177,7 +177,11 @@ public override void Simulate( Client client )
 
 			if ( !this.IsAlive() )
 			{
-				ChangeSpectateCamera();
+				if ( Prop is null )
+					ChangeSpectateCamera();
+				else
+					SimulatePossession();
```

Comment:
When the `prop` goes null we fail to change the camera to `FreeSpectatingCamera` since `CancelPossession` doesn't get called.

Possess a prop that will break (there are plenty on casino), have the prop break, `CancelPossession` doesn't get called.